x/bounty/client/cli/tx.go

```diff
@@ -183,3 +185,95 @@ func NewSubmitFindingCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewHostAcceptFindingCmd implements accept a finding by host.
+func NewHostAcceptFindingCmd() *cobra.Command {
```

Comment:
note anything after this command returns is broadcasted to the node and is public

Reply:
done